From 439675fd38eba7174cabf8abb623bd3f1c9edceb Mon Sep 17 00:00:00 2001
From: wang-bin <wbsecg1@gmail.com>
Date: Sun, 23 Dec 2018 20:09:15 +0800
Subject: [PATCH 3/3] win: fix build without d3d9

---
 src/libANGLE/renderer/d3d/RendererD3D.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/libANGLE/renderer/d3d/RendererD3D.cpp b/src/libANGLE/renderer/d3d/RendererD3D.cpp
index dc3f063bd..35b4b0d96 100644
--- a/src/libANGLE/renderer/d3d/RendererD3D.cpp
+++ b/src/libANGLE/renderer/d3d/RendererD3D.cpp
@@ -229,7 +229,9 @@ GLenum DefaultGLErrorCode(HRESULT hr)
 {
     switch (hr)
     {
+#ifdef ANGLE_ENABLE_D3D9
         case D3DERR_OUTOFVIDEOMEMORY:
+#endif
         case E_OUTOFMEMORY:
             return GL_OUT_OF_MEMORY;
         default:
-- 
2.20.1 (Apple Git-117)

