From 0afbb9b38b0357723210bc3920d858f8d8dc9f34 Mon Sep 17 00:00:00 2001
From: wang-bin <wbsecg1@gmail.com>
Date: Fri, 23 Aug 2019 13:53:32 +0800
Subject: [PATCH 5/5] clang-cl: std::atomic is not trivally constructible

---
 src/libGLESv2/global_state.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/libGLESv2/global_state.cpp b/src/libGLESv2/global_state.cpp
index 8ea912eea..2d311d657 100644
--- a/src/libGLESv2/global_state.cpp
+++ b/src/libGLESv2/global_state.cpp
@@ -36,11 +36,12 @@ namespace
 static TLSIndex threadTLS = TLS_INVALID_INDEX;
 Debug *g_Debug            = nullptr;
 std::atomic<std::mutex *> g_Mutex;
+#if !(defined(__clang__) && defined(_MSC_VER))
 static_assert(std::is_trivially_constructible<decltype(g_Mutex)>::value,
               "global mutex is not trivially constructible");
 static_assert(std::is_trivially_destructible<decltype(g_Mutex)>::value,
               "global mutex is not trivially destructible");
-
+#endif
 Thread *AllocateCurrentThread()
 {
     ASSERT(threadTLS != TLS_INVALID_INDEX);
-- 
2.21.0

